### PR TITLE
fix: Remove support for `staged_predict` in CatBoost

### PR DESCRIPTION
**Pull request by ssheng**
## Description

Supporting `staged_predict()` in CatBoost needs further discussion. Remove support from CatBoost for now.

## Checklist:

- [ ] My code follows the bentoml code style, both `make format` and
  `make lint` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: tests/integration/frameworks/models/catboost.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def check(out):
     save_kwargs={
         "signatures": {
             "predict": {"batchable": False},
-            "staged_predict": {"batchable": False},
+            # TODO: staged_predict is not supported by catboost yet
```
Comment (aarnphm) [inline tests/integration/frameworks/models/catboost.py:55]:
```suggestion
            # TODO: staged_predict is not supported by bentoml.catboost yet
```

**Comment by yetone** [comment]
thanks, @ssheng !

File: tests/integration/frameworks/models/catboost.py [hunk 2]
```diff
@@ -65,12 +65,7 @@ def check(out):
                         expected=accurate_to(y_test, 0.9466),
                     ),
                 ],
-                "staged_predict": [
-                    Input(
-                        input_args=[X_test],
-                        expected=generator_accurate_to(y_test, 0.7),
-                    ),
-                ],
+                # TODO: staged_predict is not supported by catboost yet
```
Comment (aarnphm) [inline tests/integration/frameworks/models/catboost.py:68]:
```suggestion
                # TODO: staged_predict is not supported by bentoml.catboost yet
```

**Review by aarnphm — APPROVED** [review]

**Review by aarnphm — APPROVED** [review]

**Comment by ssheng** [comment]
The PR has been merged. This PR was not closed due to Github bug.

**Pull request closed without merging**